projects/1.21/assets/jei/jei/lang/zh_cn.json

```
@@ -0,0 +1,207 @@
+    "config.jei.mode.comment": "更改JEI的操作方式。",
+    "config.jei.mode.cheatItemsEnabled": "JEI作弊模式",
+    "config.jei.mode.cheatItemsEnabled.comment": "给予物品而非显示其配方。",
+    "config.jei.mode.editEnabled": "编辑模式",
+    "config.jei.mode.editEnabled.comment": "在列表中点击物品以隐藏或取消隐藏。",
+    "config.jei.interface": "用户界面",
+    "config.jei.interface.comment": "用户界面的相关设定。",
+    "config.jei.interface.overlayEnabled": "启用物品列表",
+    "config.jei.interface.overlayEnabled.comment": "打开GUI后，在一侧显示物品列表。",
+    "config.jei.interface.bookmarkOverlayEnabled": "显示书签列表",
+    "config.jei.interface.bookmarkOverlayEnabled.comment": "打开GUI后，在一侧显示书签列表。",
+    "config.jei.search": "搜索选项",
+    "config.jei.search.comment": "搜索框的相关选项。",
+    "config.jei.search.modNameSearchMode": "@模组名称",
+    "config.jei.search.modNameSearchMode.comment": "当按模组名称搜索时（前缀：@）",
+    "config.jei.search.tooltipSearchMode": "#工具提示",
+    "config.jei.search.tooltipSearchMode.comment": "当按工具提示搜索时（前缀：#）",
+    "config.jei.search.tagSearchMode": "$标签",
+    "config.jei.search.tagSearchMode.comment": "当按标签名称搜索时（前缀：$）",
+    "config.jei.search.creativeTabSearchMode": "%%创造模式标签页",
+    "config.jei.search.creativeTabSearchMode.comment": "当按创造模式标签页名搜索时（前缀：%）",
+    "config.jei.search.colorSearchMode": "^颜色",
+    "config.jei.search.colorSearchMode.comment": "当按物品颜色搜索时（前缀：^）",
+    "config.jei.search.resourceIdSearchMode": "&资源ID",
+    "config.jei.search.resourceIdSearchMode.comment": "当按资源ID搜索时（前缀：&）",
+    "config.jei.search.searchAdvancedTooltips": "搜索高级工具提示",
+    "config.jei.advanced": "高级",
+    "config.jei.advanced.comment": "用于更改JEI功能的高级配置。",
+    "config.jei.advanced.itemBlacklist": "物品黑名单",
+    "config.jei.advanced.itemBlacklist.comment": "黑名单中的物品不会显示在JEI物品列表中。格式：modId[:name[:meta]]。使用编辑模式可自动添加或删除这里的条目。",
+    "config.jei.advanced.colorSearchEnabled": "启用颜色搜索",
+    "config.jei.advanced.colorSearchEnabled.comment": "按颜色搜索物品，并在可以此法搜索的物品的信息提示中加以注明。",
+    "config.jei.advanced.debugModeEnabled": "调试模式",
+    "config.jei.advanced.debugModeEnabled.comment": "仅对JEI开发者有用，启用后会添加测试物品以及一些调试用配方。",
+    "config.jei.advanced.centerSearchBarEnabled": "搜索框居中",
+    "config.jei.advanced.centerSearchBarEnabled.comment": "启用后，搜索框会被移动至屏幕底部居中的位置。",
+    "config.jei.advanced.modNameFormat": "模组名称格式",
+    "config.jei.advanced.modNameFormat.comment": "定义JEI界面的信息提示中的模组名称格式。留空以禁用。",
+    "config.jei.advanced.maxColumns": "最大宽度",
+    "config.jei.advanced.maxColumns.comment": "输入物品列表的最大宽度。",
+    "config.jei.advanced.maxRecipeGuiHeight": "配方界面最大高度",
+    "config.jei.advanced.maxRecipeGuiHeight.comment": "配方查询界面允许的最大高度。",
+    "config.jei.advanced.giveMode": "拿取模式",
+    "config.jei.advanced.giveMode.comment": "选择JEI是将物品直接放置于物品栏（inventory）还是用鼠标 （mouse_pickup） 抓取。",
+    "config.jei.advanced.addBookmarksToFront": "预添加新书签",
+    "config.jei.advanced.addBookmarksToFront.comment": "设为true时，将新书签添加到书签列表的前面；为false时，将新书签添加到书签列表的末尾",
+  
+    "_comment": "Hide Ingredients Mode",
+    "gui.jei.editMode.description": "JEI编辑模式：",

```

| **审查级别**    |
|:-:|
| ❓疑问 / 需要作者确认 |

**原文: JEI Hide Ingredients Mode:**

```suggestion
    "gui.jei.editMode.description": "JEI编辑模式：",
```